code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -38,6 +38,7 @@
 		"water",
 		"ethanol",
 		"sugar",
+		"sacid",
```

Comment:
.... did you readd the wrong one? I thought we talked about re-adding the welding fuel since there wasn't the dedicated wall tank on TGCode.

Reply:
Yes I did. I just had sulfuric acid on the mind for some reason.